huaweicloud/services/acceptance/live/resource_huaweicloud_live_record_callback_test.go

```diff
@@ -0,0 +1,79 @@
+package live
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/live/v1/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getRecordCallbackResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.HcLiveV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating Live v1 client: %s", err)
+	}
+	return client.ShowRecordCallbackConfig(&model.ShowRecordCallbackConfigRequest{Id: state.Primary.ID})
+}
+
+func TestAccRecordCallback_basic(t *testing.T) {
+	var obj model.ShowRecordCallbackConfigResponse
+
+	pushDomainName := fmt.Sprintf("%s.huaweicloud.com", acceptance.RandomAccResourceNameWithDash())
+	rName := "huaweicloud_live_record_callback.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getRecordCallbackResourceFunc,
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testCallBack_basic(pushDomainName, "http://mycallback.com.cn/record_notify"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "domain_name", pushDomainName),
+					resource.TestCheckResourceAttr(rName, "type.#", "1"),
```

Comment:
For the TypeList parameter, we can check each element, just a tip.

Reply:
done